### PR TITLE
Added ENSA and Sagasser Brands

### PR DESCRIPTION
**Pull request by gabemanfroi**
Added ENSA to office/insurance.json and Sagasser to shop/beverages.json
edit: fix issues #3570 and #3565

### CODE REVIEW

**Comment by gabemanfroi** [comment]
> One advice to make life easier of maintainers in github: when a pull request fixes an issue, add a line like "fix issue #3570" so automatically the mentioned issue is notified and can be closed.

Copy that! Sorry for the inconvenience and thanks for the advice!

**Comment by bmillemathias** [comment]
> Sorry for the inconvenience

Please, no worries.

also when I wrote
> when a pull request fixes an issue, add a line like ...

I meant
> when a commit fixes an issue, add a line like ...

**Comment by bhousel** [comment]
> also when I wrote
> 
> > when a pull request fixes an issue, add a line like ...
> 
> I meant
> 
> > when a commit fixes an issue, add a line in the commit like ...

I think it actually works both ways!
GitHub is pretty clever.
https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

File: brands/office/insurance.json [hunk 1]
```diff
@@ -725,5 +725,15 @@
       "name:fa": "بیمه ایران",
       "office": "insurance"
     }
+  },
+   "office/insurance|ENSA": {
+    "countryCodes": ["ao"],
+    "tags": {
+      "brand": "ENSA – Seguros de Angola S.A.",
```
Comment (bhousel) [inline brands/office/insurance.json]: Just "ENSA" is probably fine here.
```suggestion
      "brand": "ENSA",
```

File: brands/office/insurance.json [hunk 2]
```diff
@@ -725,5 +725,15 @@
...
+      "brand:wikidata": "Q5323726",
+      "brand:wikipedia": "en:ENSA_–_Seguros_de_Angola",
```
Comment (bhousel) [inline brands/office/insurance.json]: Need to switch underscores for spaces in the `brand:wikipedia` tag..
```suggestion
      "brand:wikipedia": "en:ENSA – Seguros de Angola",
```

File: brands/shop/beverages.json [hunk 3]
```diff
@@ -135,5 +135,15 @@
...
       "shop": "beverages"
     }
+  },
+  "shop/beverages|Sagasser":{
+    "countryCodes": ["de"],
+    "tags": {
+      "brand": "Sagasser",
+      "brand:wikidata":"Q30838417" ,
+      "brand:wikipedia": "de:Sagasser", 
+      "name": "SAGASSER-Vertriebs GmbH",
```
Comment (bhousel) [inline brands/shop/beverages.json]: Just "Sagasser" is fine here..
```suggestion
      "name": "Sagasser",
```

**Comment by gabemanfroi** [comment]
> Thanks @gabemanfroi and welcome 👋
> Just a few minor requests..
> 
> * We need to replace the underscores with spaces in the `brand:wikipedia` tag
> * This project tends to use the _familiar_ names for things - i.e. what a person would call the place in conversation or expect to see on a rendered map.  OSM doesn't really map "legal names" although there is a separate tag for `official_name` for situations where a business has another longer name that it goes by, and people just want to capture it for some reason.

Thanks for your support.
I really appreciate the attention and the advices! Thank you again!

**Pull request merged**